octoprint_slic3r/profile.py

```diff
@@ -239,11 +261,18 @@ def to_slic3r_ini(cls, profile, path, display_name=None, description=None):
 	@classmethod
 	def convert_value(cls, key, value, default, sep=","):
 		try:
-			if key in float_or_percentage:
+			'''Some defaults can be instances of int whereas its values are instances of float. 
```

Comment:
This should be a comment, not a doc string. The problem should also possibly rather addressed by making the defaults floats (e.g. `1.0` instead of just `1`). I might not be understanding the issue correctly though.